File: pinyougou-web-seckill/src/main/webapp/js/service/seckillGoodsService.js
//服务层
app.service('seckillGoodsService',function($http){
//读取列表数据绑定到表单中
    this.findList=function(){
        //alert("22");
        return $http.get('seckillGoods/findList.do');
    }


    this.findOne=function(id){
        //alert("22");
        return $http.get('seckillGoods/findOne.do?id='+id);
    }
});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/seckill/SeckillGoodsService.java
package cn.itcast.core.service.seckill;

import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.List;

public interface SeckillGoodsService {

    /**
     * 查询全部符合条件的秒杀商品
     * @return
     */
    List<SeckillGoods> findList();


    SeckillGoods findOne(Long id);
}
